packages/verifiable-credentials/src/credentials.ts

```diff
@@ -88,9 +88,25 @@ export default class Credentials {
 	 *
 	 * @param {string} vcJwt
 	 */
-	async verifyCredential(vcJwt: string): Promise<unknown> {
+	async verifyCredential(vcJwt: string): Promise<any> {
 		const resolver = this.getResolver();
-		return verifyCredential(vcJwt, resolver);
+		const decodedCredential = await verifyCredential(vcJwt, resolver);
+
+		if (decodedCredential) {
+			const payload = decodedCredential.payload
+			const vc = payload.vc
+
+			if (vc.expirationDate) {
+				// Ensure credential hasn't expired
+				const now = new Date().toISOString()
+				if (vc.expirationDate < now) {
```

Comment:
This comparison is potentially incorrect. Both `.toISOString()` and the `XMLSCHEMA11-2` can include non-UTC timezone offsets, and this is just doing a string comparison. 

A case like this can give the incorrect result:

"2019-08-29T04:00:00+00:00:00" > "2019-08-29T04:00:00+02:00:00"

Not sure what the best solution is here - maybe convert to UTC? 

@tahpot may have suggestions.

Reply:
Ah yes. Javascript `toIsoString()` is always UTC, but the W3C spec allows `expirationDate` to be other timezones.

Moment.js is deprecated. [See a list of options here](https://momentjs.com/docs/#/-project-status/).

We need something that is as lightweight as possible. Maybe Luxon. https://moment.github.io/luxon/#/?id=luxon

@nick-verida Happy for you to make a call here on the best option.

Reply:
@cmcWebCode40 The Luxon library linked above supports [ISO8601](https://moment.github.io/luxon/#/parsing?id=iso-8601) parsing and this is compatible with `XMLSCHEMA11-2` date objects. 

So use that.

Reply:
https://github.com/verida/verida-js/issues/147#issue-1138053083

Reply:
@cmcWebCode40 That backlog issue isn't correct.

The required task is to use the Luxon library to do the date/time comparison.

Reply:
update the backlog task